.prettierignore

```diff
@@ -7,4 +7,5 @@ lib
 node_modules
 examples
 test/**/*.json
-test/**/*.yaml
\ No newline at end of file
+test/**/*.yaml
+test/e2e/testdata
```

Comment:
Need to ignore code formatting for the test configuration data otherwise the HTTP recordings won't precisely match.